### PR TITLE
Add precious metals alloy recipe for ABS

### PR DESCRIPTION
**Pull request by lc-1337**

### CODE REVIEW

**Comment by chochem** [comment]
which part of the spreadsheet is used? 0.8, 1, or 1.2?

**Comment by lc-1337** [comment]
0.8

**Comment by lc-1337** [comment]
Any comments about balance? I can manually make recipes for better balance/exclude some materials

**Comment by Dream-Master** [comment]
@lordIcocain add a recipe is one thing but remove temp limit from auto generation is a NO  for me

File: src/main/java/gtPlusPlus/xmod/gregtech/loaders/RecipeGen_BlastSmelterGT_GTNH.java [hunk 1]
```diff
@@ -154,10 +173,13 @@ public static synchronized boolean generateGTNHBlastSmelterRecipesFromEBFList()
                 time = x.mDuration;
                 enabled = x.mEnabled;
                 special = x.mSpecialValue;
-
-                // continue to next recipe if the Temp is too high.
-                if (special > 3600) {
-                    Logger.MACHINE_INFO("[ABS] Skipping ABS addition for GTNH due to temp.");
+                // black list apply
```
Comment (Dream-Master) [inline src/main/java/gtPlusPlus/xmod/gregtech/loaders/RecipeGen_BlastSmelterGT_GTNH.java]: NO

**Review by Dream-Master — DISMISSED** [review]

**Comment by Pxx500** [comment]
> I remove temp limit from auto generation, now more mid/high tier materials can be turned in molten state without extra steps.
> 
> I made spreadsheet, so you can evaluate changes in balance. https://docs.google.com/spreadsheets/d/1UcUOfZ-YLdZry5sUNt2flKcjokm9x58p9p0uWdW9S_w/edit?usp=sharing
> 
> Also added one hand made recipe for molten precious metals alloy, same reason, get molten without extra steps

please add more information to the spreadsheet, are the numbers seconds? what is the before/after?
this excel looks like it's some kind of backend, no explanation whatsoever about what anything means

**Comment by lc-1337** [comment]
Bump

**Comment by Dream-Master** [comment]
need to check ingame

**Review by chochem — APPROVED** [review]
that one recipe is fine.

**Pull request merged**